static.config.js

```diff
@@ -115,43 +120,43 @@ export default {
 
 
     return [
+      ...allPages(),
       {
         path: '/',
         component: 'src/js/pages/Home',
-        // getProps: () => ({
-        //   allServicePages: serviceQueries['en'],
-        // }),
+        getData: async () => ({
+          allServicePages: topServices,
```

Comment:
We'll need to add topServices query (or kill the x-mas tree page) to homepage requests in all languages. Currently the homepage in spanish, vietnamese, etc experience the same initial load failure.

Reply:
Thanks for catching this. I got excited that changing the query fixed things but forgot to take it across the finish line for the other languages.

Reply:
fixed this with my pr feedback commit.